### PR TITLE
Enhancement: introducing the juggler!

### PR DESCRIPTION
**Pull request by till**
* add a new release to the opcache
 * remove the old one
 * add a note on that a reset is considered harmful

easybib/issues#4597

### CODE REVIEW

File: src/EasyBib/OPcache/Juggler.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        }
+
+        if (1 === $primer->doPopulate()) {
+            return self::ERROR_NEW_RELEASE;
+        }
+
+        $primer->setPath($oldRelease);
+        if (1 === $primer->validate()) {
+            return self::ERROR_OLD_RELEASE;
+        }
```
Comment (glaubinix) [inline src/EasyBib/OPcache/Juggler.php]: none of the 3 calls currently returns 1.

**Comment by glaubinix** [comment]
@till this looks good. It might be useful to add a note that this only works if you do ````composer dumpautoload --optimize````.

**Comment by till** [comment]
@glaubinix want to look again? If this works, I'll send out a PR tomorrow to Academy.

**Comment by glaubinix** [comment]
@till as far as I can tell this looks ok.

**Pull request merged**